modules/ROOT/pages/mule-standalone.adoc

```diff
@@ -4,13 +4,11 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, studio, enterprise, ee, premium features, paid features, purchase, license, licensed
 
-You only need to install Mule, a production-strength runtime engine, if you host it in your own environment.
+If you want to host Mule runtime engine in your own environment, xref:hardware-and-software-requirements.adoc[verify that you meet hardware and software requirements], then xref:runtime-installation-task.adoc[download and install Mule].
 
-You don't need to worry about installation in the following instances:
+You don't need to install Mule in the following scenarios:
 
-* You use Studio for development and testing, because Studio contains Mule.
-* MuleSoft hosts your instance of Mule.
+* When you use Studio for development and testing, because Studio contains Mule embedded.
```

Comment:
You don't really need the second half
```suggestion
* When you use Anypoint Studio for development and testing
```
But if you want it, consider
```suggestion
* When you use Anypoint Studio for development and testing, because Studio contains Mule
```
OR
```suggestion
* When you use Anypoint Studio (which contains Mule) for development and testing
```